component.json

```diff
@@ -253,6 +253,52 @@
           "prompt": "Please select a Event object"
         }
       }
+    },
+    "streamPlatformEventsPubSub": {
+      "title": "Subscribe to PubSub",
+      "order": 97,
+      "help": {
+        "description": "Can be used for subscription to the specified in the configuration Platform Event object. Can be used only for Realtime flows",
+        "link": "/components/salesforce/triggers#subscribe-to-platform-events-trigger"
+      },
+      "main": "./lib/triggers/streamPlatformEventsPubSub.js",
+      "type": "polling",
+      "fields": {
+        "object": {
+          "viewClass": "SelectView",
+          "label": "Event object name",
+          "required": true,
+          "model": "objectTypes",
+          "prompt": "Please select a Event object"
+        },
+        "pubsubEndpoint": {
+          "label": "Pub/Sub API Endpoint",
+          "viewClass": "TextFieldView",
+          "required": false,
+          "placeholder": "api.pubsub.salesforce.com:7443",
+          "help": {
+            "description": "You can set Pub/Sub API Endpoint manually or leave it blank for default: \"api.pubsub.salesforce.com:7443\""
+          }
+        },
+        "eventCountPerRequest": {
+          "label": "Number of events per request",
```

Comment:
Are there any upper limits?

Reply:
OK. Found this in the code 'Number of events per request" must be valid number between 1 and 100'. It makes sense to write it somewhere here

Reply:
done